web/src/lib/components/pages/record/profile-actions.tsx

```diff
@@ -0,0 +1,84 @@
+import { h, Component } from 'preact';
+import Profile from '../profile/profile';
+import User from '../../../user';
+
+interface Props {
+  user: User;
+}
+
+interface State {
+  profileFormVisible: boolean;
+  whyProfileVisible: boolean;
+}
+
+export default class ProfileActions extends Component<Props, State> {
+  state: State = {
+    profileFormVisible: false,
+    whyProfileVisible: false,
+  };
+
+  private toggleProfileForm = () => {
+    this.setState({ profileFormVisible: !this.state.profileFormVisible });
+  };
+
+  private toggleWhyProfile = () => {
+    this.setState({ whyProfileVisible: !this.state.whyProfileVisible });
+  };
```

Comment:
@mikehenrty has been using the approach of declaring instance methods, then `.bind()`ing them in the constructor, instead of assigning lambdas to fields.

i dont much prefer either variant, but IMO we should try to be consistent here. please talk to @mikehenrty about this :)

Reply:
Thanks, good point. I prefer class properties, since they're less noisy. But I'll hash it out with @mikehenrty.